pkgs/development/python-modules/requests/default.nix

```
@@ -17,7 +17,7 @@
 
 buildPythonPackage rec {
   pname = "requests";
-  version = "2.32.3";
+  version = "2.32.4";
   format = "setuptools";
```

```suggestion
  pyproject = true;
```
and set `build-system = [ setuptools ]`